src/Constants.js

```diff
@@ -21,6 +21,7 @@ export const ENTITY_TYPE = {
   LINK: 'LINK',
   IMAGE: 'IMAGE',
   INPUT: 'INPUT',
+  VIDEO: 'VIDEO',
```

Comment:
Let's change this to `IFRAME` instead of `VIDEO` because it might be any kind of content really.